Makefile

```diff
@@ -57,13 +55,14 @@ build-dep-fedora:
 		python3-pytest \
 		python3-requests \
 		python3-types-requests \
+		python3-setuptools_scm \
 		rpmdevtools \
 		tar
 
 .PHONY: rpm
 rpm: $(RPM)
 
-$(RPM): $(generated)
+$(RPM): aiven/client/version.py
```

Comment:
Everything related to RPM should be dropped in my opinion. It's not the correct way of building RPMs. If we want to make RPMs for everybody, we should use [OBS](https://openbuildservice.org/) or something similar.

Reply:
does it maybe make sense also to remove `test-fedora` step from tests?

Reply:
Why? We use Fedora downstream, so tests won't hurt.